portal/assets/themes/unsplash/custom_portal.js

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Maybe you want to remove or empty this file instead of having just comment?

Reply:
thougt it was in standard skeleton therefore compulsory, it's removed in `5428b7f`